exporter/trace/trace_proto.go

```diff
@@ -115,11 +116,14 @@ func protoFromSpanSnapshot(s *export.SpanSnapshot, projectID string, format Disp
 	if s.ParentSpanID != s.SpanContext.SpanID && s.ParentSpanID.IsValid() {
 		sp.ParentSpanId = s.ParentSpanID.String()
 	}
-	if s.StatusCode != codes.Ok {
-		sp.Status = &statuspb.Status{Code: int32(s.StatusCode), Message: s.StatusMessage}
-	}
-	if s.StatusCode == codes.Ok && s.StatusMessage != "" {
-		sp.Status = &statuspb.Status{Code: int32(s.StatusCode), Message: s.StatusMessage}
+	if s.StatusCode == codes.Ok {
+		sp.Status = &statuspb.Status{Code: int32(codepb.Code_OK)}
+	} else if s.StatusCode == codes.Unset {
```

Comment:
Looks like when `s.StatusCode == nil` it would hit the `else` block, but it should go here.

Reply:
This isn't a pointer, so it can't be nil.

Reply:
oops didn't notice that. I guess the Go SDK defaults to `Unset`